package.json

```diff
@@ -10,6 +10,9 @@
     "lint": "eslint .",
     "depcheck": "depcheck",
     "start": "NODE_ENV=production node app/index.js",
+    "start:persona:01": "NODE_ENV=production API_SWAGGER_BG_COLOR='#e4f2f3' API_SWAGGER_TITLE=IdentityAPIheidi     API_SWAGGER_HEADING='Identity API System - heidi'     node app/index.js",
+    "start:persona:02": "NODE_ENV=production API_SWAGGER_BG_COLOR='#e7f6e6' API_SWAGGER_TITLE=IdentityAPIemma      API_SWAGGER_HEADING='Identity API System - emma'      node app/index.js",
+    "start:persona:03": "NODE_ENV=production API_SWAGGER_BG_COLOR='#f8dddd' API_SWAGGER_TITLE=IdentityAPIreginald  API_SWAGGER_HEADING='Identity API System - reginald'  node app/index.js",
```

Comment:
I don't think these should be separate scripts and these certainly shouldn't be in this repo as it doesn't exclusively work with hyproof. 

When you do a similar PR to https://github.com/digicatapult/dscp-hyproof-api we should update the `docker-compose-3-persona.yml`

Reply:
Makes sense. DONE!